containers/sandbox/Dockerfile

```diff
@@ -19,3 +19,9 @@ RUN apt-get update && apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
 RUN service ssh start
+
+RUN pip install jupyterlab notebook jupyter_kernel_gateway
+# Add common data science utils
+RUN pip install transformers[torch]
+RUN pip install torch --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Do you expect agent would run `torch` to solve user's problems?

Reply:
Yes! I did try to use CodeActAgent-Mistral 7B and let it write Pytorch code for some simple tasks (e.g., regression), and it did actually solve the problem by 80% - I suspect by switching towards a stronger base model (e.g., 70B), the agent will be able to tackle such problems, so i'd like to keep these as in the container.

alternatively in the future if we are concerned about the size of the container, we can make these optional and/or allow user to use their customized executor image.

Reply:
I don't really have a concrete qualitative example at hand using PyTorch, the closest example I have is having the agent/LLM using SKLearn for machine learning tasks: https://twitter.com/xingyaow_/status/1754556862949994917

Reply:
that's cool!